scripts/build.sh

```diff
@@ -18,6 +20,21 @@ then
   done
 fi
 
+## compute modules that were modified in this PR
+if [ -z "$modules" -a -n "$PR" ]
+then
+  modified=`curl -s https://api.github.com/repos/vaadin/vaadin-flow-components/pulls/$PR/files \
+    | jq -r '.[] | .filename' | grep 'vaadin.*parent' | perl -pe 's,^vaadin-(.*)-flow-parent.*,$1,g' | sort -u`
+  if [ `echo "$modules" | wc -w` -lt 5 ]
```

Comment:
Won't `$modules` be empty here?

Reply:
Because modules is empty, and it's wrapped in quotes it will output zero here which is correct try this : `echo "" | wc -w`

Reply:
you're right, it should be `$modified` fixed